flashinfer_bench/cli/main.py

```
@@ -439,6 +477,65 @@ def cli():
+        help="Seconds between dispatch and reconciliation passes.",
+    )
+    router_parser.add_argument(
+        "--max-attempts",
+        type=int,
+        default=5,
+        help="Maximum infrastructure dispatch attempts per task.",
+    )
+    router_parser.add_argument(
+        "--result-ttl", type=int, default=86_400, help="Seconds to retain terminal task responses."
+    )
+    router_parser.add_argument("--host", type=str, default="0.0.0.0", help="Router host")
```

Ruff (0.16.2)

[error] Possible binding to all interfaces

(S104)